### PR TITLE
Added language tags

### PR DESCRIPTION
**Pull request by MeuhV1**

### CODE REVIEW

**Comment by MeuhV1** [comment]
I replaced all "No Translation" texts with the English text. Only for non-pedia text entries.

I used a script to auto-replace everything. It should have replaced only the new tags I created in the first commit.

**Comment by MeuhV1** [comment]
Added translation for vanilla techs and quotes

File: Assets/XML/Text/DynamicNames/DynamicNames_Arabia.xml [hunk 1]
```diff
@@ -304,14 +340,26 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_ARABIAN_NAME_SPAIN</Tag>
 		<English>Esbanya</English>
+		<French>banya</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Arabia.xml]: Looks like something went wrong here.

File: Assets/XML/Text/DynamicNames/DynamicNames_Arabia.xml [hunk 2]
```diff
@@ -340,6 +388,10 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_ARABIAN_MUGHALS</Tag>
 		<English>Emirate of Delhi</English>
+		<French>mirate of Delhi</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Arabia.xml]: Here too.

File: Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml [hunk 3]
```diff
@@ -36,70 +44,138 @@
...
 		<English>Nicaea</English>
+		<French>Nicaea</French>
+		<German>Nicaea</German>
+		<Italian>Nicaea</Italian>
+		<Spanish>Nicaea</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTIUM_EPIRUS</Tag>
 		<English>Epirus</English>
+		<French>pirus</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml]: .

File: Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml [hunk 4]
```diff
@@ -36,70 +44,138 @@
...
 		<English>Exarchate of %s1</English>
+		<French>xarchate of %s1</French>
+		<German>xarchate of %s1</German>
+		<Italian>xarchate of %s1</Italian>
+		<Spanish>xarchate of %s1</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTINE_EGYPT</Tag>
 		<English>Exarchate of Alexandria</English>
+		<French>xarchate of Alexandria</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml]: .

File: Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml [hunk 5]
```diff
@@ -36,70 +44,138 @@
...
+		<German>xarchate of Alexandria</German>
+		<Italian>xarchate of Alexandria</Italian>
+		<Spanish>xarchate of Alexandria</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTINE_BABYLONIA</Tag>
 		<English>Exarchate of Ctesiphon</English>
+		<French>xarchate of Ctesiphon</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml]: .

File: Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml [hunk 6]
```diff
@@ -36,70 +44,138 @@
...
+		<German>xarchate of Ctesiphon</German>
+		<Italian>xarchate of Ctesiphon</Italian>
+		<Spanish>xarchate of Ctesiphon</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTINE_GREECE</Tag>
 		<English>Exarchate of Achaia</English>
+		<French>xarchate of Achaia</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml]: .

File: Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml [hunk 7]
```diff
@@ -36,70 +44,138 @@
...
+		<German>xarchate of Achaia</German>
+		<Italian>xarchate of Achaia</Italian>
+		<Spanish>xarchate of Achaia</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTINE_CARTHAGE</Tag>
 		<English>Exarchate of Africa</English>
+		<French>xarchate of Africa</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml]: .

File: Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml [hunk 8]
```diff
@@ -36,70 +44,138 @@
...
+		<German>xarchate of Africa</German>
+		<Italian>xarchate of Africa</Italian>
+		<Spanish>xarchate of Africa</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTINE_PERSIA</Tag>
 		<English>Exarchate of Parthia</English>
+		<French>xarchate of Parthia</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml]: .

File: Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml [hunk 9]
```diff
@@ -36,70 +44,138 @@
...
+		<German>xarchate of Parthia</German>
+		<Italian>xarchate of Parthia</Italian>
+		<Spanish>xarchate of Parthia</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTINE_ROME</Tag>
 		<English>Exarchate of Ravenna</English>
+		<French>xarchate of Ravenna</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml]: .

File: Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml [hunk 10]
```diff
@@ -36,70 +44,138 @@
...
+		<German>xarchate of Ravenna</German>
+		<Italian>xarchate of Ravenna</Italian>
+		<Spanish>xarchate of Ravenna</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIV_BYZANTINE_SPAIN</Tag>
 		<English>Exarchate of Spania</English>
+		<French>xarchate of Spania</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Byzantium.xml]: .

File: Assets/XML/Text/DynamicNames/DynamicNames_Carthage.xml [hunk 11]
```diff
@@ -4,23 +4,38 @@
...
 	<TEXT>
 		<Tag>TXT_KEY_CIV_PHOENICIA_ADJECTIVE</Tag>
 		<English>Phoenician</English>
+		<French>Phoenician</French>
 		<German>
 			<Text>Ph&#246;nizisch:ph&#246;nizischen:ph&#246;nizische</Text>
 			<Gender>Neuter:Neuter:Neuter</Gender>
 			<Plural>0:0:0</Plural>
 		</German>
+		<Italian>No Translation</Italian>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Carthage.xml]: Should use English text.

File: Assets/XML/Text/DynamicNames/DynamicNames_China.xml [hunk 12]
```diff
@@ -69,6 +81,7 @@
 			<Gender>Female</Gender>
 			<Plural>1</Plural>
 		</French>
+		<German>No Translation</German>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_China.xml]: Should use English text.

File: Assets/XML/Text/DynamicNames/DynamicNames_Colombia.xml [hunk 13]
```diff
@@ -40,21 +52,41 @@
...
 		<English>Falangist %s1</English>
+		<French>Falangist %s1</French>
+		<German>Falangist %s1</German>
+		<Italian>Falangist %s1</Italian>
+		<Spanish>Falangist %s1</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIV_COLOMBIA_EMPIRE</Tag>
 		<English>Empire of Gran Colombia</English>
+		<French>mpire of Gran Colombia</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Colombia.xml]: .

File: Assets/XML/Text/DynamicNames/DynamicNames_Japan.xml [hunk 14]
```diff
@@ -81,6 +81,7 @@
 			<Gender>Male</Gender>
 			<Plural>0</Plural>
 		</French>
+		<German>No Translation</German>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Japan.xml]: Should use English text.

File: Assets/XML/Text/DynamicNames/DynamicNames_Mexico.xml [hunk 15]
```diff
@@ -33,6 +37,7 @@
 			<Gender>Male</Gender>
 			<Plural>1</Plural>
 		</French>
+		<German>No Translation</German>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Mexico.xml]: Should use English text.

File: Assets/XML/Text/DynamicNames/DynamicNames_Moors.xml [hunk 16]
```diff
@@ -100,26 +124,50 @@
...
 		<English>Andalusian Taifas</English>
+		<French>Andalusian Taifas</French>
+		<German>Andalusian Taifas</German>
+		<Italian>Andalusian Taifas</Italian>
+		<Spanish>Andalusian Taifas</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIV_MOORISH_ARABIA</Tag>
 		<English>East Umayyad Caliphate</English>
+		<French>ast Umayyad Caliphate</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Moors.xml]: .

File: Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml [hunk 17]
```diff
@@ -116,10 +132,18 @@
...
 		<English>Khedivate of %s1</English>
+		<French>Khedivate of %s1</French>
+		<German>Khedivate of %s1</German>
+		<Italian>Khedivate of %s1</Italian>
+		<Spanish>Khedivate of %s1</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIV_OTTOMAN_BABYLONIA</Tag>
 		<English>Eyalet of Mesopotamia</English>
+		<French>yalet of Mesopotamia</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml]: .

File: Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml [hunk 18]
```diff
@@ -172,6 +196,10 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_OTTOMAN_PHOENICIA</Tag>
 		<English>Eyalet of Syria</English>
+		<French>yalet of Syria</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml]: .

File: Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml [hunk 19]
```diff
@@ -200,6 +228,10 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_OTTOMAN_BYZANTIUM</Tag>
 		<English>Eyalet of R&#251;m</English>
+		<French>yalet of R&#251;m</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml]: .

File: Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml [hunk 20]
```diff
@@ -228,10 +260,18 @@
 	<TEXT>
 		<Tag>TXT_KEY_CIV_OTTOMAN_RUSSIA</Tag>
 		<English>Eyalet of Yedisan</English>
+		<French>yalet of Yedisan</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml]: .

File: Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml [hunk 21]
```diff
@@ -228,10 +260,18 @@
...
+		<German>yalet of Yedisan</German>
+		<Italian>yalet of Yedisan</Italian>
+		<Spanish>yalet of Yedisan</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_CIV_OTTOMAN_INDONESIA</Tag>
 		<English>Eyalet of Aceh</English>
+		<French>yalet of Aceh</French>
```
Comment (dguenms) [inline Assets/XML/Text/DynamicNames/DynamicNames_Ottomans.xml]: .

File: Assets/XML/Text/External/AutoSave.xml [hunk 22]
```diff
@@ -5,49 +5,65 @@
...
 		<English>When checked, a saved game is created after (re)generating the map.</English>
+		<French>When checked, a saved game is created after (re)generating the map.</French>
 		<German>Erzeugt einen Spielstand, wenn eine Karte (neu-)erstellt wurde</German>
 		<Italian>Quando &#232; selezionato, viene creato un salvataggio dopo la rigenerazione della mappa.</Italian>
+		<Spanish>When checked, a saved game is created after (re)generating the map.</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_BUG_OPT_AUTOSAVE__CREATEENDSAVE_TEXT</Tag>
 		<English>End Save</English>
+		<French>d Save</French>
```
Comment (dguenms) [inline Assets/XML/Text/External/AutoSave.xml]: .

File: Assets/XML/Text/External/AutoSave.xml [hunk 23]
```diff
@@ -5,49 +5,65 @@
...
 		<English>When checked, a saved game is created when the game is won/lost.</English>
+		<French>When checked, a saved game is created when the game is won/lost.</French>
 		<German>Erzeugt einen Spielstand, wenn eine Partie gewonnen/verloren wird</German>
 		<Italian>Quando &#232; selezionato, viene creato un salvataggio quando viene vinta/persa la partita.</Italian>
+		<Spanish>When checked, a saved game is created when the game is won/lost.</Spanish>
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_BUG_OPT_AUTOSAVE__CREATEEXITSAVE_TEXT</Tag>
 		<English>Exit Save</English>
+		<French>xit Save</French>
```
Comment (dguenms) [inline Assets/XML/Text/External/AutoSave.xml]: .

File: Assets/XML/Text/External/BUFFY.xml [hunk 24]
```diff
@@ -11,155 +11,209 @@
 		<French>Activ&#233;</French>
 		<German>Aktivieren</German>
 		<Italian>Attivato</Italian>
+		<Spanish>abled</Spanish>
```
Comment (dguenms) [inline Assets/XML/Text/External/BUFFY.xml]: .

File: Assets/XML/Text/External/Event Signs.xml [hunk 25]
```diff
@@ -13,14 +13,18 @@
 	<TEXT>
 		<Tag>TXT_KEY_BUG_OPT_EVENTSIGNS__ENABLED_TEXT</Tag>
 		<English>Event Signs</English>
+		<French>vent Signs</French>
```
Comment (dguenms) [inline Assets/XML/Text/External/Event Signs.xml]: .

File: Assets/XML/Text/External/Reminder.xml [hunk 26]
```diff
@@ -8,47 +8,54 @@
 		<French>Activer les Rappels [ALT + M]</French>
 		<German>Erinnerungen aktivieren</German>
 		<Italian>Abilita i Promemoria [ALT + M]</Italian>
+		<Spanish>able Reminders [ALT + M]</Spanish>
```
Comment (dguenms) [inline Assets/XML/Text/External/Reminder.xml]: .

File: Assets/XML/Text/External/Strategy Overlay.xml [hunk 27]
```diff
@@ -10,140 +10,185 @@
 		<French>Activ&#233;</French>
 		<German>Aktivieren</German>
 		<Italian>Attivata</Italian>
+		<Spanish>abled</Spanish>
```
Comment (dguenms) [inline Assets/XML/Text/External/Strategy Overlay.xml]: .

**Review by dguenms — CHANGES_REQUESTED** [review]
Some English texts should have been used.

There is also an issue with texts starting with a capital E, seems like the regex did not capture those correctly. I only reviewed diffs small enough to show up, so you should probably do a search for "<English>E" and verify for the entire diff.

**Comment by MeuhV1** [comment]
It seems my script removed the first letters of all entries starting with an E.

Fixed it.

**Review by dguenms — COMMENTED** [review]
There are still some comments open where the English text should be used.

**Comment by MeuhV1** [comment]
Replaced all "No translations" text with the English texts. (Except pedia texts, they still have the "No translation" text)

Merged all "translation" commits into 1 commit.

**Review by dguenms — APPROVED** [review]
If you remove the recent conflicts I can merge this.

**Comment by MeuhV1** [comment]
Done.

**Pull request merged**